<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minecraft Server Checker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Minecraft.png</normaloff>Minecraft.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 23, 23);</string>
  </property>
  <widget class="QFrame" name="MainFrame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Server_infos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ServerIP_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>1</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Server IP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServerIP_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Input Server IP here.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Check_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);</string>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLabel" name="Ping_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ping_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ServerSoftware_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Server's Minecraft Version:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServerSoftware_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Status_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Status_display">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ServerMOTD_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MOTD:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServerMOTD_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Server_players">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Players_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Players :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Players_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="MinecraftServerChecker">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Minecraft Server Checker</string>
    </property>
   </widget>
   <widget class="QLabel" name="Version_text">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>v0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MinecraftLogo_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Minecraft Server Checker | About</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Minecraft.png</normaloff>Minecraft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="ColorSideFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>31</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
